src/widgets/_windows.scss

```diff
@@ -5,6 +5,7 @@ dialog {
     > decoration {
         border-radius: rem(6px);
         margin: rem(12px);
+        transition: all 90ms ease-in-out;
```

Comment:
I'm getting really mixed/choppy framerates here even on decently powerful hardware. I don't know if this is worth doing unless it's really smooth.